reduxApp2/dev/js/index.js

```diff
@@ -0,0 +1,25 @@
+import 'babel-polyfill';
+import React from 'react';
+import ReactDOM from "react-dom";
+import {Provider} from 'react-redux';
+import {createStore, applyMiddleware} from 'redux';
+import thunk from 'redux-thunk';
```

Comment:
@VARTIKASINGH1234 Remove the package thunk and promise. and please remove all the unnecessary imports

Reply:
removed all unnecessary imports.